### PR TITLE
Prop types menu button

### PR DESCRIPTION
**Pull request by Glazy**
Add `propTypes` for `menu-button` and update the associated documentation with a new `required` column for each prop.

Gets some way towards closing: #35 🎉

### CODE REVIEW

**Comment by ryanisinallofus** [comment]
Is the idea to just add a column here for #35?

screen shot 2018-09-04 at 5 08 50 pm

**Comment by Glazy** [comment]
Yeah @ryanisinallofus :+1:

File: packages/menu-button/src/index.js [hunk 1]
```diff
@@ -228,6 +229,15 @@ let MenuListImpl = React.forwardRef(
   )
 );
 
+MenuListImpl.propTypes = {
```
Comment (ryanflorence) [inline packages/menu-button/src/index.js]: do we need these since this is just an implementation detail?

**Comment by ryanflorence** [comment]
lovely! thank you

**Pull request merged**